lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -752,6 +752,21 @@ bool selectionSupportsShading()
     return false;
 }
 
+#ifdef UFE_V3_FEATURES_AVAILABLE
```

Comment:
I think this function is unused and came back during the conflict resolution.

Reply:
I see it used at line 1211. The Linux and OS/X builds do not allow an unused function to exist. I agree it should not have suddenly appeared from conflict resolution since it was added a month ago in PR #2762 and should have been in the branch I started with. This is strange.

Reply:
Not strange finally. The code was added last month in a branch that was merged yesterday. This is why it appeared in as a conflict to resolve in my branch.